cspell.config.json

```diff
@@ -8,7 +8,8 @@
     "lefthook",
     "configurated",
     "noopener",
-    "noreferrer"
+    "noreferrer",
+    "hola"
```

Comment:
Please remove this and use a inline comment to avoid issues with cspell:

`cspell:disable-next`